pkg/service/init.go

```diff
@@ -322,7 +322,9 @@ func RunE2E(clientset *kubernetes.Clientset) {
 
 // Cleanup removes all resources created during E2E tests.
 func Cleanup(clientset *kubernetes.Clientset) {
+	common.SetDefaultNamespace()
 	namespace := viper.GetString("namespace")
+	fmt.Println("using namespace:", namespace)
```

Comment:
use log instead of fmt see https://github.com/kubernetes-sigs/hydrophone/blob/main/pkg/common/args.go#L76 for example